node/README.md

```diff
@@ -1,9 +1,11 @@
-![CI](https://github.com/env0/env0-client-integrations/workflows/CI/badge.svg?branch=master)
-
-# env0 NodeJS CLI
 
+# @env0/cli ![](https://img.shields.io/npm/l/@env0/cli?color=blue) ![](https://img.shields.io/npm/v/@env0/cli) ![](https://img.shields.io/npm/dw/@env0/cli) ![](https://github.com/env0/env0-client-integrations/workflows/CI/badge.svg?branch=master) 
```

Comment:
when you click on the badge you should go to npm

Reply:
@yaronya currently all badges doesn't have links. 

1. CI should point to github actions - https://github.com/env0/env0-client-integrations/actions?query=workflow%3ACI
2. licence bade should point to the license page - https://github.com/env0/env0-client-integrations/blob/master/LICENSE
3. NPM badge and download badge should point to the NPM - https://www.npmjs.com/package/@env0/cli